lib/commands/context.js

```diff
@@ -184,6 +191,35 @@ helpers.startChromedriverProxy = async function startChromedriverProxy (context)
       if (androidPackage && androidPackage.length > 0) {
         opts.chromeAndroidPackage = androidPackage[1];
       }
+      if (!opts.extractChromeAndroidPackageFromContextName) {
+        if (_.has(this.opts, 'enableWebviewDetailsCollection') && !this.opts.enableWebviewDetailsCollection) {
+          // When enableWebviewDetailsCollection capability is explicitly disabled, try to identify
+          // chromeAndroidPackage based on contexts, known chrome variant packages and queryAppState result
+          // since webviewsMapping does not have info object
+          const contexts = webviewsMapping.map((wm) => wm.webviewName);
+          for (const knownPackage of KNOWN_CHROME_PACKAGE_NAMES) {
+            if (_.includes(contexts, `${WEBVIEW_BASE}${knownPackage}`)) {
+              continue;
+            }
+            const appState = await this.queryAppState(knownPackage);
+            if (_.includes([APP_STATE_RUNNING_IN_BACKGROUND, APP_STATE_RUNNING_IN_FOREGROUND], appState)) {
+              opts.chromeAndroidPackage = knownPackage;
+              log.debug(`Identified chromeAndroidPackage as '${opts.chromeAndroidPackage}'` +
```

Comment:
a single space is missing here

Reply:
Thanks, done as https://github.com/appium/appium-android-driver/pull/699/commits/add55a1245710d1117f089216d9af1bdee747052